### ISSUE TITLE
Create a `Reviewers: Admin` group and `Reviews:Edit` permission

### ISSUE THREAD

**Issue by jvillalobos** [mozilla/addons-server#7279]
Part of #7172

The `Reviews:Edit` permission will be used to gate admin reviewer tools. The `Reviewers: Admin` group will have this permission and should include the following people:
* Jorge (ID: 4230)
* Andreas (ID: 4750720)
* Philipp (ID: 85036)
* Caitlin (ID: 12642224)

/cc @wagnerand and @kewisch to determine who if any of the review contractors should be in this group.

**Comment by wagnerand** [comment]
Could you summarize what features will use that permission?

**Comment by jvillalobos** [comment]
Should be the admin reviewer cases presented in the [PRD](https://docs.google.com/document/d/1Me-2Qx-C7cMptKQt_-1gAyPZCBOiww9laQLEMlD9N2M/edit): access all reviewer tools, undelete user ratings, handle admin-flagged add-ons, add/remove users in reviewer groups, setting the MOTD, edit canned responses, disable add-ons,...

Flipping the question, are there specific tasks that contractor reviewers should be able to do that regular reviewers shouldn't? There might be room for a different group or permission.

**Comment by wagnerand** [comment]
> Flipping the question, are there specific tasks that contractor reviewers should be able to do that regular reviewers shouldn't?

Absolutely, handling admin-flagged add-ons is a good example for that. I went through the PRD again and looked at all user-stories for admin reviewers. In order for contractors to do their work, they need this permission.

Given the sensitivity and the impact, I would split all group member management tasks into a separate permission, which would be given to the four people you mentioned in the initial comment, in order to minimize risk.

**Comment by jvillalobos** [comment]
Okay. Group management isn't planned for this phase anyway, so this issue can continue as-is.

**Comment by jvillalobos** [comment]
No, that's not on the PRD at the moment.

**Comment by diox** [comment]
Note: I am renaming `Reviews:Edit` permission to `Reviews:Admin` in https://github.com/mozilla/addons-server/pull/7347

**Comment by AlexandraMoga** [comment]
Verified on AMO-dev, FF57 (Win10)

I've seen that a a new reviewer's group with the required permission has been set-up in Admin tools, However, the user I have included in this group doesn't seem to have access at all in Reviewer Tools. Is this access limited on -dev as well only to the  users mentioned above?

![image](https://user-images.githubusercontent.com/31961530/35277900-ed561bde-0050-11e8-84d6-16cfa3524479.png)

**Comment by diox** [comment]
Your user will need to be part of another group to access reviewer tools themselves. The permission, and therefore the group, only gives access to some specific admin-only things inside reviewer tools.

**Comment by AlexandraMoga** [comment]
@jvillalobos could you please confirm what @diox mentioned above to know exactly what was expected here?

**Comment by jvillalobos** [comment]
That's correct, I'm trying to reduce permission overlap going forward.

**Comment by AlexandraMoga** [comment]
Thanks! Since this is the expected behavior, I'm marking this as verified fixed.
